defaults/main.yml

```diff
@@ -27,6 +27,28 @@ gitlab_version: '8.16'
 # GitLab will be configured with HTTPS enabled by default
 gitlab_domain: [ 'code.{{ ansible_domain }}' ]
 
+# .. envvar:: gitlab_pages_domain
+#
+# The main domain served by Gitlab Pages.
+#
+# This should be a different one than ``gitlab_domain`` to prevent cross domain cookie attacks.
+gitlab_pages_domain: ''
```

Comment:
Is `gitlab-pages` support active by default? If not, set it as disabled and only enable if `gitlab_pages_domain` is non-empty. I think that would be best because GitLab Pages requires additional configuration in the DNS, custom wildcard certificates, etc.